client/ayon_core/cli_commands.py

```diff
@@ -149,36 +143,35 @@ def publish(path: str, targets: list=None, gui:bool=False) -> None:
         log.info("Publish finished.")
 
     @staticmethod
-    def extractenvironments(output_json_path, project, asset, task, app,
-                            env_group):
+    def extractenvironments(
+        output_json_path, project, asset, task, app, env_group
+    ):
         """Produces json file with environment based on project and app.
 
         Called by Deadline plugin to propagate environment into render jobs.
         """
 
-        from ayon_core.lib.applications import (
-            get_app_environments_for_context,
-            LaunchTypes,
-        )
+        from ayon_core.addon import AddonsManager
 
-        if all((project, asset, task, app)):
-            env = get_app_environments_for_context(
-                project,
-                asset,
-                task,
-                app,
-                env_group=env_group,
-                launch_type=LaunchTypes.farm_render
```

Comment:
This PR changed this launch type from `LaunchTypes.farm_render` to `LaunchTypes.farm_publish` for all calls to `extractenvironments` CLI - including Deadline Render jobs which I suppose is an incorrect change?

Reported as issue here: https://github.com/ynput/ayon-applications/issues/2